recipe/meta.yaml

```diff
@@ -1,52 +1,57 @@
 {% set name = "diff-cover" %}
-{% set version = "7.4.0" %}
+{% set version = "9.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/diff_cover-{{ version }}.tar.gz
-  sha256: 3d98800ea3198ab8450733050dd98dec802c9028c7164aace990f2f13a160e98
+  sha256: c6c9f7f1eacd9019d118cafb75908445caa41a79f65f36f9f506c9626dbcbc70
 
 build:
-  noarch: python
   number: 0
+  skip: true  # [py<38]
   entry_points:
     - diff-cover = diff_cover.diff_cover_tool:main
     - diff-quality = diff_cover.diff_quality_tool:main
-  script: '{{ PYTHON }} -m pip install . --no-deps -vv '
+  script: '{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv '
 
 requirements:
   host:
     - pip
-    - poetry
-    - python >=3.7.2
+    - poetry-core >=1.0.7
+    - python
+    - wheel
```

Comment:
```suggestion
```
`wheel` is redundant if `poetry-core` is a build backend because poetry-core has its own WheelBuilder, see https://github.com/python-poetry/poetry-core/blob/8c878fd4a81b526dc8d91fcd5f7406ece37d9202/src/poetry/core/masonry/builders/wheel.py#L51